modules/release-notes/pages/relnotes.adoc

```diff
@@ -547,7 +547,7 @@ WARNING: Changing the internal replication setting will cause all XDCR processes
 | *Summary*: Introduced a delay in updating the RBAC database revision number to avoid returning AUTH_STALE during a refresh of the RBAC database.
 
 | https://issues.couchbase.com/browse/MB-32497[MB-32497^]
-| *Summary*: Connection::signalIfIdle may cause a deadlock when used from a different thread
+| *Summary*: Memcached connections may hang due to `Connection::signalIfIdle` triggering a deadlock when used from a different thread
```

Comment:
Ah sorry, I missed this rewording: this is better, but I think it's still under the fixed issues section? So better worded as what has been fixed?